package.json

```diff
@@ -18,10 +18,11 @@
   },
   "homepage": "https://github.com/elastic/eslint-config-kibana#readme",
   "peerDependencies": {
-    "eslint": "3.11.1",
-    "eslint-plugin-mocha": "4.7.0",
-    "eslint-plugin-babel": "4.0.0",
-    "eslint-plugin-react": "^6.10.3",
-    "babel-eslint": "6.1.2"
+    "babel-eslint": "^7.2.3",
```

Comment:
Yeah, kibana will take care of pinning in a couple days (via yarn.lock). No reason for that to continue changing how to list deps in our libraries.